### PR TITLE
Miracle Changes

### PR DESCRIPTION
**Pull request by EvenInDeathIStillServe**
Lesser Miracle has been renamed to Miracle.
Miracle has been renamed to Fortify.

Miracle healing is now a 10 second over-time effect. It comes with a little red cross effect to indicate when its active.
Fortify will increase all healing during a 1 minute period by a factor of x1.5.

There is no longer a need to target a specific limb for the healing to take place.

Decreased the range on both spells from 7 (full screen) to 4 tiles.

Really unrelated: makes windows not have more integrity than metal bars.

### CODE REVIEW

**Comment by BlindPoet** [comment]
waow..

**Comment by Ollanius** [comment]
I like how the new miracles feel.
With that being said, I have two things I'd like to mention here:

- Miracles no longer heal CRIPPLED limbs, which is - from what I understand - an unintentional change. If you could restore that capability, that'd be very nice.
- Miracles no longer heal BLEEDING limbs. This is an interesting change, and it does make it much more important for people to pack needles-and-cloth with them. Not sure if it's intentional or not, but I'd like to retain this for a bit longer to see how it plays out.

**Comment by EvenInDeathIStillServe** [comment]
> Good change, but consider making the red cross sprites golden instead?
> 
> Feel like it would fit the religious theme better and also IIRC there are ethical concerns with using the red cross in video games.

Yeah it's just a placeholder cause my ass can't sprite for shit.

File: code/datums/status_effects/rogue/roguebuff.dm [hunk 1]
```diff
@@ -230,3 +230,43 @@
...
+	H.color = "#FF0000"
+	owner.adjustBruteLoss(-healing_on_tick)
+	owner.adjustFireLoss(-healing_on_tick)
+	var/list/wCount = owner.get_wounds()
+	if(owner.blood_volume < BLOOD_VOLUME_NORMAL)
+		owner.blood_volume = min(owner.blood_volume+10, BLOOD_VOLUME_NORMAL)
+	if(wCount.len > 0)
+		owner.heal_wounds(healing_on_tick)
+		owner.update_damage_overlays()
+	owner.adjustBruteLoss(-healing_on_tick, 0)
```
Comment (Ephemeralis) [inline code/datums/status_effects/rogue/roguebuff.dm:256]: is it intentional that you're doubling up on brute+fire healing here? lines 248 and 249 already seem to do this which makes me think not

Comment (EvenInDeathIStillServe) [reply code/datums/status_effects/rogue/roguebuff.dm:256]: Ack. My bad. Will fix when I get home.

Comment (EvenInDeathIStillServe) [reply code/datums/status_effects/rogue/roguebuff.dm:256]: Ack. My bad. Will fix when I get home.

**Pull request merged**